Docs/Configuration.tex

```diff
@@ -6108,23 +6108,23 @@ \subsection{Quirks Properties}\label{uefiquirkprops}
 
   This is an experimental quirk, which should only be used for the aforementioned problem.
   In all other cases the quirk may render the operating system unstable and is not recommended.
-  The recommended solution in the other cases is to install a kernel driver like
+  The recommended solution in the other cases is to install a kernel driver such as
   \href{https://github.com/RehabMan/VoodooTSCSync}{VoodooTSCSync},
   \href{https://github.com/interferenc/TSCAdjustReset}{TSCAdjustReset},
   or \href{https://github.com/lvs1974/CpuTscSync}{CpuTscSync} (a more specialised
   variant of VoodooTSCSync for newer laptops).
 
   \emph{Note}: The reason this quirk cannot replace the kernel driver is
-  because it cannot operate in ACPI S3 mode (sleep wake) and because the UEFI firmwares
-  provide very limited multicore support preventing the precise update of the MSR
+  because it cannot operate in ACPI S3 mode (sleep wake) and because the UEFI firmware
+  provides very limited multicore support preventing the precise update of the MSR
   registers.
 
 \item
   \texttt{UnblockFsConnect}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: Some firmwares block partition handles by opening them
-  in By Driver mode, which results in File System protocols being unable to install.
+  \textbf{Description}: Some types of firmware block partition handles by opening them
+  in \texttt{By\ Driver} mode, resulting in being unable to install File System Protocols.
```

Comment:
Protocols should probably not be capitalised as it is not part of the name.

Reply:
The UEFI spec is a bit inconsistent, but generally uses capital case for UEFI Protocols. Lower case is usually used for network protocols etc., but not always.

Reply:
Probably can go either way. Will drop to lowercase. 
I used uppercase because of "File System" ... Was the actual intention "Filesystem"